NEXT_CHANGELOG.md

```diff
@@ -26,6 +26,17 @@ By [@USERNAME](https://github.com/USERNAME) in https://github.com/apollographql/
 # [x.x.x] (unreleased) - 2022-mm-dd
 
 ## ❗ BREAKING ❗
+
+### Fix the supported defer specification version to 20220824 ([PR #1652](https://github.com/apollographql/router/issues/1652))
+
+Since the router will ship before the @defer specification is done, we
+add a parameter to the Accept and Content-Type headers to indicate
+which specification version is accepted.
```

Comment:
```suggestion
Since the router will ship before the `@defer` specification is done, we
add a parameter to the `Accept` and `Content-Type` headers to indicate
which specification version is accepted.
```